### ISSUE TITLE
blend luxcore messes meshes up in blender 3.5.a

### ISSUE THREAD

**Issue by SlincHD** [LuxCoreRender/BlendLuxCore#809]
Steps to reproduce: start rendering in blender

Screenshots or rendered images: 
![image](https://user-images.githubusercontent.com/85231739/215327470-aea7ce74-3fa1-44f5-8aeb-76563685efc5.png)

OS: Windows 10
Blender version: 3.5.a
LuxCore version: 2.6.1

**Comment by neo2068** [comment]
Yes, Blender seems to have changed mesh data accessibility between 18.12.2022, commit 8709a51fa907 and now commit 042775ad4829. Last time I checked it worked normally. Version 3.5 is not released officially and we have no ressources to support the fast changing interfaces of this development version. We will adapt the interface when there is a release candidate. I will keep this issue open to keep track of the broken functionallity of the interface.

**Comment by SlincHD** [comment]
yes i don't need a fix now just letting you guys know that it exsist for the future and thank you for keeping luxcore alive!

**Comment by neo2068** [comment]
For further investigation, this bug is most likely caused by this commit in blender code: https://github.com/blender/blender/commit/1af62cb3bf46f0cd328ab0840a1363eca938c628

**Comment by SlincHD** [comment]
today was lounch of 3.5! just letting you know :3 King

**Comment by neo2068** [comment]
I am working on the fix for Blender 3.5. There are several breaking changes in the python API which have to be addressed.

- [x] vertex position is now located under attributes
- [ ] uv maps are broken
- [ ] vertex colors are broken

**Comment by neo2068** [comment]
The initial bug has been resolved. I will close this ticket and work on the problem with modifiers in ticket #822.
